src/main/java/ca/wolfram/beta/symath/operations/MultiplyNode.java

```diff
@@ -45,7 +91,15 @@ public double operationEval(VMap map) {
     public String toString() {
         StringBuilder s = new StringBuilder();
         s.append("(");
-        getChildren().forEach((c) -> s.append(c).append(" * "));
-        return s.substring(0, s.length() - 3) + ")";
+        boolean isFirst = true;
+        for (MathNode c : getChildren()) {
+            if (isFirst) {
+                s.append(c.toString());
+                isFirst = false;
+                continue;
+            }
+            s.append(" * ").append(c.toString());
+        }
```

Comment:
I think it's best we still have it print () just so we know that this node exists. This will also help us identify errors, because having 0 children means it should simplify into nothing.